app/models/organization.rb

```diff
@@ -18,4 +18,8 @@ class Organization < ApplicationRecord
   }
 
   has_many :donations, dependent: :destroy
+
+  def profile_image
```

Comment:
why does this need to belong as an instance method, if all we're doing is returning a string?? shouldn't this be a constants thing?

Reply:
it was a place holder. still need to replace with avatar_url

Reply:
I left a comment to move this method to a helper since user model has it too.